test-harness/modules/mocker/endpoints/marathon.py

```diff
@@ -409,3 +423,9 @@ def break_leader_reply(self, *_):
         """
         with self._context.lock:
             self._context.data["leader-content"] = 'blah blah buh buh'
+
+    def _reset(self):
```

Comment:
Would it be a problem for you to refactor all other endpoints this way as well - lets be consistent.